app/models/space.rb

```diff
@@ -61,4 +61,13 @@ def jitsi_meet_domain
 
     Utilities.from_utility_hookup(jitsi_hookup).meet_domain
   end
+
+  attr_accessor :blueprint
+  after_create :apply_blueprint, if: -> { blueprint.present? }
+
+  def apply_blueprint
```

Comment:
To summarize our discussion during ensemble: let's move this into a factory method like `Space.create_with_blueprint`, then adjust the controller action to use this new method when a blueprint is provided.